extensions/remote-repository/README.md

```diff
@@ -0,0 +1,108 @@
+# @loopback/remote-repository
+
+Access models persisted by a remote LoopBack application.
+
+## Stability: ⚠️Experimental⚠️
+
+> Experimental packages provide early access to advanced or experimental
+> functionality to get community feedback. Such modules are published to npm
+> using `0.x.y` versions. Their APIs and functionality may be subject to
+> breaking changes in future releases.
+
+## Installation
+
+```sh
+npm install --save @loopback/remote-repository
+```
+
+## Basic use
+
+### Initial setup
+
+Create a new DataSource using
+[openapi connector](https://github.com/strongloop/loopback-connector-openapi),
+configure it to connect to your remote LoopBack application.
+
+**IMPORTANT: You must configure the datasource to use named parameters. Answer
+the prompt _Use positional parameters instead of named parameters?_ with _n_.**
+
+Copy model files from your remote LoopBack application to your local project. We
```

Comment:
Does this mean to copy( or import)  LB3/LB4 model from the remote LoopBack app to the local project in a LB4 form?

Reply:
Great question!

If the remote application is written in LoopBack 4, then the model files can be copied over or perhaps shared via an npm package.

If the remote application is written in LoopBack 3, then there are few options available:
1. Import the model(s) from the remote LB3 app to the local LB4 app via `lb4 import-lb3-models`.
2. Use a tool like `lb4 openapi --client` to generate LB4 models (or interfaces?) describing the shape of model data from the Swagger spec provided by the LB3 app.

In both cases, it's important to consider how to update local models after the remote application has changed them? For example, when the remote application adds a new property `updatedAt`, how to update the local model to include that new property too?

- When LB4 models are shared via a copy, then we need to copy the new version again.
- When LB4 models are shared via a npm package, then we need to update local project's dependencies.
- When LB3 models are imported via `lb4 import-lb3-models`, we need to re-run that command again.
- When LB3 models are imported via `lb4 openapi --client`, we need to re-run the import process.